paasta_tools/cli/cmds/status.py

```diff
@@ -1864,10 +1871,11 @@ def print_kafka_status(
             PaastaColors.green if broker["phase"] == "Running" else PaastaColors.red
         )
 
+        # NOTE: the k8s API returns timestamps in UTC, so we make sure to always work in UTC
         start_time = datetime.strptime(
             broker["deployed_timestamp"], "%Y-%m-%dT%H:%M:%SZ"
-        )
-        delta = datetime.utcnow() - start_time
+        ).replace(tzinfo=timezone.utc)
```

Comment:
this will add a +00:00 to broker started times - e.g., we'll go from 
```
Started
2024-09-18 01:11:36 (20 hours ago)
```

to

```
Started
2024-09-18 01:11:36+00:00 (20 hours ago)
```

but imo, this is a net positive since without looking at the parenthetical, it's entirely unclear what TZ is being used